rollup.config.js

```diff
@@ -17,8 +17,6 @@ import pkg from './package.json';
 import inputs from './exports';
 
 const APP_VERSION_STRING = '__uikit_app_version__';
```

Comment:
nit - small suggestion, can we:
```suggestion
const APP_VERSION_STRING = '__react_dev_mode__';
```

This will be the string's value in storybook ~

Reply:
Addressed in https://github.com/sendbird/sendbird-uikit-react/pull/551/commits/3e234c9551e74889e3d8b2cb804062947056a3ac